src/client/common.py

```diff
@@ -397,7 +468,12 @@ def curlify(service, req_str, gnucli=False, execute=False, prettyprint=True):
 
     if execute:
         request_string = requestify(service_url, params, verb)
-
+        if windowsEncryptedPassword:
+            password_json = do_request(verb, request_string, to_print=False)
```

Comment:
A try catch block would be good. Else if the backend response changes, the client would throw weird responses.